logic/minecraft/onesix/OneSixWrite.cpp

```diff
@@ -179,6 +181,6 @@ QJsonDocument OneSixFormat::toJson(PackagePtr file, bool saveOrder)
 QJsonObject OneSixFormat::toJson(JarmodPtr mod)
 {
 	QJsonObject out;
-	writeString(out, "name", mod->name);
```

Comment:
Guess what happens if I revert this? (it's possible that it was fixed after Json.{h,cpp} got included in unstable, I'll check later)